docs/dev-docs/conf.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why do you add a a dev-docs directory - what do you plan to put there? Is this a mistake?

Reply:
I'll take a look but I suspect it was there before

Reply:
Looking at docs/ today, there is no docs/dev-docs.